### DISCUSSION TITLE
Clear Kimai Cache with softaculous

### DISCUSSION THREAD

**Discussion by danieldrinhausen** [kimai/kimai#4616] [Questions & Answers]
Hey guys,

I recently installed Kimai on my webspace via a 1 Click Installer (no ssh access). What is the best way to clear cache?

Thanks in advance.

Cheers,
Daniel

**Comment by kevinpapst** [comment]
Ask Softaculous. I would strongly recommend to NOT use that. 
AFAIK they do not offer updates or anything else that is required to run Kimai successfully long term.

**Reply by danieldrinhausen** [reply]
Thanks Kevin! So better get some webspace that supports SSH and install it this way? I would also be able to update Kimai by SSH?

**Reply by kevinpapst** [reply]
Please read the docs:
https://www.kimai.org/documentation/installation.html
https://www.kimai.org/documentation/updates.html
https://www.kimai.org/documentation/shared-hosting.html
